### PR TITLE
Separate fixed selectors from the witness

### PR DESCRIPTION
**Pull request by volhovm**
With this PR, fixed selectors are not anymore part of the witness, and are passed separately. The intention is that they must be static for a given circuit.

### CODE REVIEW

File: msm/src/prover.rs [hunk 1]
```diff
@@ -417,6 +482,16 @@ where
         .map(|poly| (coefficients_form(poly), hiding(1)))
         .collect();
 
+    // @volhovm: I'm not sure we need to prove opening of fixed
```
Comment (dannywillems) [inline msm/src/prover.rs:485]: They should be treated as "common preprocessed input" from PlonK: https://eprint.iacr.org/2019/953.pdf, page 27.

**Comment by dannywillems** [comment]
@volhovm I messed up with the merge, can you push force the old version, please?

**Comment by dannywillems** [comment]
Oh, merging previous PR fixed the mess I made. Fixing conflicts again :)

**Comment by volhovm** [comment]
@dannywillems merged the branch = fixed conflicts. Feel free to review. If you feel like there are more things to do I can do another PR round.

**Review by dannywillems — APPROVED** [review]
Accepting to move on. However, we should have an "index" or a setup phase as specified before.

**Pull request merged**